tools/firtool/firtool.cpp

```diff
@@ -700,6 +700,11 @@ processBuffer(MLIRContext &context, TimingScope &ts, llvm::SourceMgr &sourceMgr,
     pm.nest<firrtl::CircuitOp>().nest<firrtl::FModuleOp>().addPass(
         firrtl::createMergeConnectionsPass(mergeConnectionsAgggresively));
 
+  // Lower the ref.resolve and ref.send ops and remove the RefType ports.
+  // LowerToHW cannot handle RefType so, this pass must be run to remove all
+  // RefType ports and ops.
+  pm.nest<firrtl::CircuitOp>().addPass(firrtl::createLowerXMRPass());
+
```

Comment:
Might be nice to preserve this for the `-ir-fir` path, maybe sink into the conditional below for paths that do some lowering?

Reply:
Thanks for catching that, was a mistake :P